src/cli/test/test_cli.c

```diff
@@ -3246,14 +3246,6 @@ static void test_trn_cli_update_transit_network_policy_subcmd(void **state)
 				  "cidr_ip": "172.0.0.9",
 				  "cidr_type": "1",
 				  "bit_value": "10"
-			  },
-			  {
```

Comment:
As suggested by @phudtran and @Hong-Chang during the team meeting, please see if removing test cases is the right fix or if we should fix the code to handle multiple policy update items.